bond/bond.py

```diff
@@ -401,7 +442,8 @@ def get_all_metadata_fields(self):
         return sorted(found_fields)
 
     def remove_metadata_fields(self, fields_to_remove):
-        """Removes specific fields from all metadata files."""
+        '''Removes specific fields from all metadata files.'''
```

Comment:
Why the switches to single quotes? I think double quotes are more common for docstrings